default.py

```diff
@@ -1,12 +1,15 @@
 #-*- coding: utf-8 -*-
 import sys, getopt
 import datetime
-from checks import *
 from models import Sites, SiteTests
 import config
 import gettext
 _ = gettext.gettext
 
+TEST_ALL = -1
+
+(TEST_UNKNOWN_01, TEST_GOOGLE_LIGHTHOUSE, TEST_PAGE_NOT_FOUND, TEST_UNKNOWN_03, TEST_UNKNOWN_04, TEST_UNKNOWN_05, TEST_HTML, TEST_CSS, TEST_UNKNOWN_08, TEST_UNKNOWN_09, TEST_UNKNOWN_10, TEST_UNKNOWN_11, TEST_UNKNOWN_12, TEST_UNKNOWN_13, TEST_UNKNOWN_14, TEST_UNKNOWN_15, TEST_UNKNOWN_16, TEST_UNKNOWN_17, TEST_UNKNOWN_18, TEST_UNKNOWN_19, TEST_WEBBKOLL) = range(21)
```

Comment:
I assume this is used to translate numbers representing tests to something a little more readable? I would prefer getting rid of the numbers instead, but I assume that would be a lot of work. (or make it more cumbersome to use...)

I had no idea variables could be defined this way. Clever but it took me a while to get! Maybe add a comment explaining what's going on and why (especially with the `TEST_UNKNOWN...` stuff.)

It's a very long line, could be helpful to wrap, either by some set limit, or after every `,`

Reply:
This is exactly why I changed to this a long time ago (It was now only moved from checks.py).
A loong time ago it was hardcoded numbers repeated in different places.
It was very hard to read and know if this was the page not found test or not.
We use numbers because the orginal implementation had it this way and webperf.se is using this field on the website.
It is also used when you want to only run one test (see below screenshot regarding the).
<img width="675" alt="Screenshot 2020-11-30 at 19 10 20" src="https://user-images.githubusercontent.com/4742476/100647450-d38b2580-333f-11eb-8c3f-fa02d685f6a0.png">

Regarding the TEST_UNKNOWN strings you have to ask @marcusosterberg as why he keept them when making the WebPerf_Core public :)
I only noticed there where test there that was not included in the core library and made it possible to add them in the future.

My suggestion is that you add a seperate issue regarding documenting this (I agree that it could and should be documentet better) as it is not a change releated to this ticket.

Sounds good?